UniWatch/Controllers/ClassController.cs

```diff
@@ -88,7 +87,7 @@ public ActionResult Create([Bind(Include = "Name,Number,Section,Semester,Year")]
         public ActionResult Delete(int classId)
```

Comment:
As a Teacher I couldn't get there. I kept getting redirected to the login page. I added the authorization attribute and I was able to get to this page.